swarm/tools/lint_routing_fields.py

```
@@ -244,12 +257,13 @@ def should_skip_file(file_path: Path) -> bool:
 
 
 def check_file(
-    file_path: Path, check_new: bool = False
+    file_path: Path, root: Path, check_new: bool = False
```

Ruff (0.15.2)

[error] Boolean-typed positional argument in function definition

(FBT001)